cmd/image-builder/main.go

```diff
@@ -409,13 +411,11 @@ func cmdDescribeImg(cmd *cobra.Command, args []string) error {
 }
 
 func run() error {
-	// images generates a lot of noisy logs a bunch of stuff to
-	// Debug/Info that is distracting the user (at least by
-	// default, like what repos being loaded)
+	// Dependency library 'images' generates a lot of noisy logs a bunch of stuff to
+	// Debug/Info that is distracting the user (at least by default, like what repos being loaded).
 	//
-	// Disable for now until we can filter out the usless log
-	// messages.
-	logrus.SetOutput(io.Discard)
+	// Forward to the optional logger that can be turned on via options.
+	logrus.SetOutput(olog.Default().Writer())
```

Comment:
Do we still need this? Could we just remove logrus entirely?

Reply:
I will go ahead today and create a PR in images too, then depending on the order of merges I will update this. Or if it is okay to see some noisy logrus output for a while, I can remove this straight away.

Reply:
Since I already filed a PR for images, I am removing this line and `logrus` completely from this project, adding this commented-out line (can be uncommented once the images PR is merged):

```go
	if verbose {
		olog.SetDefault(log.New(os.Stderr, "", 0))
		// XXX: add once images has olog support
		//images_log.SetDefault(log.New(os.Stderr, "", 0))
	}
```

Reply:
Great!

Reply:
Aha, thanks! I missed that without it we get all the verbosity from "images". I guess we can just make sure images gets merged first and then we update.